src/encoded/search.py

```diff
@@ -149,9 +150,12 @@ def search(context, request, search_type=None, return_generator=False, forced_ty
         else:
             result['@graph'] = list(graph)
             return result
-    else:
-        result['@graph'] = list(graph)
-        return result
+
```

Comment:
Grab permissions/actions (which would include "Add") for current request/auth'd user vs collection of Item type we're on.

TODO(?): Make more performant?